src/Apps/Artist/Routes/Overview/Components/ArtworkFilter/index.tsx

```diff
@@ -305,39 +305,36 @@ class Filter extends Component<Props> {
                         )}
                       </Media>
                       <Media greaterThan="xs">
-                        <Sidebar width="25%" mr={2}>
-                          <Filters />
-                        </Sidebar>
+                        <Filters />
```

Comment:
One to keep in mind, dimensions relative to parent size should go _outside_ the `Media` component.

Reply:
That's going to cause a lot of bugs 😅

Reply:
I presume these overhead issues can be smoothed out?